tests/dummy/app/router.js

```diff
@@ -17,12 +17,17 @@ Router.map(function() {
     this.route('horizontal-scroll-view');
     this.route('date-pickers', function () {
       this.route('mobile');
+      this.route('pickadate');
     });
     this.route('horizontal-list-swipers', function () {
       this.route('sly');
       this.route('gesture');
       this.route('gesture2');
     });
+    this.route('slots-selections', function () {
+      this.route('single');
+      this.route('multi');
+    })
```

Comment:
Do 'single' and 'multi' route exist, couldn't see in the PR / repo?

Reply:
not used. But I thought I could leave them, in case it's needed at some point. U guys notice everything right ? :) Let me know if I need to remove thanks

Reply:
Usually PR won't have unused or future stuff as dev looking at it can get confused. Would be good to remove or a comment `Not implemented` upto you

Reply:
ok removing then

Reply:
removed thanks